wp-content/themes/goonj-crm/functions.php

```diff
@@ -227,12 +227,7 @@ function goonj_handle_user_identification_form() {
 		// If the user does not exist in the Goonj database
 		// redirect to the volunteer registration form.
 		$volunteer_registration_form_path = sprintf(
-			'/volunteer-registration/form/#?email=%s&phone=%s&message=%s&Volunteer_fields.Which_activities_are_you_interested_in_=%s&source=%s',
-			$email,
-			$phone,
-			'not-inducted-volunteer',
-			'9', // Activity to create collection camp.
-			$source
+			'/collection-camp/volunteer-with-intent/',
 		);
```

Comment:
Is this specific only to the Collection Camp, or does this change impact other flows as well? Since we are using this in other flows too, please check it once.